wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceValidator.java

```diff
@@ -49,26 +46,16 @@
   private static final Pattern VOLUME_NAME = Pattern.compile("[a-z][a-z0-9]{1,18}");
   private static final Pattern VOLUME_PATH = Pattern.compile("/.+");
 
-  private final WorkspaceRuntimes runtimes;
```

Comment:
It does not affect anything but please remove `WorkspaceRuntimes` from `WorkspaceValidator` too
https://github.com/eclipse/che/blob/2fb979bbe1cf2fc5f4179c2440bfd4de9be73749/wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceValidatorTest.java#L46